molecule/packages-dynamic/group_vars/all

```diff
@@ -8,4 +8,4 @@ pulp_install_plugins:
 pulp_settings:
   secret_key: secret
   content_origin: "https://{{ ansible_fqdn }}"
-pulp_pkg_repo: "https://fedorapeople.org/groups/katello/releases/yum/nightly/pulpcore/el{{ ansible_distribution_major_version }}/x86_64/"
+pulp_pkg_repo: "https://yum.theforeman.org/pulpcore/3.6/$releasever/$basearch/"
```

Comment:
```suggestion
pulp_pkg_repo: "https://yum.theforeman.org/pulpcore/3.6/el{{ ansible_distribution_major_version }}/$basearch/"
```

$releasever is "different", we just have `elX` as directory here.

Reply:
You're right, I wish there was a var for it.